app/routes/events/components/EventEditor/PoolSuggestions.tsx

```diff
@@ -0,0 +1,151 @@
+import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { useState } from 'react';
+import Pill from 'app/components/Pill';
+import styles from './EventEditor.css';
+import type { SelectInput } from 'app/components/Form';
+
+type GroupSuggestion = {
+  label: string;
+  values: { label: string; value: number }[];
+  children?: number[]; // Child group ids of the suggestion (these will be removed if this suggestion is selected)
+};
+
+const suggestions: GroupSuggestion[] = [
+  {
+    label: 'Alle studenter',
+    values: [{ label: 'Students', value: 14 }],
+    children: [15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26],
+  },
+  {
+    label: 'Data',
+    values: [{ label: 'Datateknologi', value: 15 }],
+    children: [16, 17, 18, 19, 20],
+  },
+  {
+    label: 'Cybdat',
+    values: [{ label: 'Kommunikasjonsteknologi', value: 21 }],
+    children: [22, 23, 24, 25, 26],
+  },
+  {
+    label: '1. klasse',
+    values: [
+      { label: '1. klasse Datateknologi', value: 16 },
+      { label: '1. klasse Kommunikasjonsteknologi', value: 22 },
+    ],
+  },
+  {
+    label: '2. klasse',
+    values: [
+      { label: '2. klasse Datateknologi', value: 17 },
+      { label: '2. klasse Kommunikasjonsteknologi', value: 23 },
+    ],
+  },
+  {
+    label: '3. klasse',
+    values: [
+      { label: '3. klasse Datateknologi', value: 18 },
+      { label: '3. klasse Kommunikasjonsteknologi', value: 24 },
+    ],
+  },
+  {
+    label: '4. klasse',
+    values: [
+      { label: '4. klasse Datateknologi', value: 19 },
+      { label: '4. klasse Kommunikasjonsteknologi', value: 25 },
+    ],
+  },
+  {
+    label: '5. klasse',
+    values: [
+      { label: '5. klasse Datateknologi', value: 20 },
+      { label: '5. klasse Kommunikasjonsteknologi', value: 26 },
+    ],
+  },
+];
+
+type PoolSuggestionType = React.ComponentProps<
+  typeof SelectInput<{ label: string; value: number }>
+>['SuggestionComponent'];
```

Comment:
Could we maybe just export the `SuggestionComponent`-type from `SelectInput.tsx`?

Reply:
Excellent idea, updated now